qutip/solver/countstat.py

```diff
@@ -89,7 +89,7 @@ def _noise_direct(L, wlist, rhoss, J_ops):
     current = np.zeros(N_j_ops)
     noise = np.zeros((N_j_ops, N_j_ops, len(wlist)))
 
-    tr_op = tensor([qeye(n) for n in L.dims[0][0]])
+    tr_op = tensor(qeye(L.dims[0][0]))
```

Comment:
```suggestion
    tr_op = qeye(L.dims[0][0])
```
Can we remove the tensor here? This was pretty odd code to start with.